test/lib/utils/addLiquidityHelper.ts

```diff
@@ -275,15 +274,13 @@ export function assertAddLiquiditySingleToken(
         (t) => t.address !== poolState.address,
     );
 
-    const expectedQueryOutput: Omit<
-        AddLiquidityQueryOutput,
-        'amountsIn' | 'bptIndex'
-    > = {
+    const expectedQueryOutput: Omit<AddLiquidityQueryOutput, 'bptIndex'> = {
         // Query should use same bpt out as user sets
         bptOut: TokenAmount.fromRawAmount(
             bptToken,
             addLiquidityInput.bptOut.rawAmount,
         ),
+        amountsIn: addLiquidityQueryOutput.amountsIn,
```

Comment:
If I understood correctly `expectedQueryOutput` is being tested against `addLiquidityQueryOutput` so maybe using values from `addLiquidityQueryOutput` itself isn't the best?

Reply:
This is just filling out content so I don't need create a separate scenario on `getCheck` without amountsIn.
There's no amountsIn provided as input that needs checking for Add Liquidity Single Token.
Do you think that's fine? Or is it too misleading?

Reply:
But as I understand it `expectedQueryOutput` is what we expect the query to return, even in the SingleToken case query, amountsIn contains an amount and we should really be testing that against an expect amount right?

Reply:
AddLiquiditySingleToken is an exactOut transaction, so user input is actually bptOut and that is being tested against the query output.
But I agree that if that's causing confusion right now, it will definitely cause confusion on any eventual maintenance months from now.
I added a small refactor that I think will help better understand the nuances there.
Let me know what you think